tests/oshiftroutetests/oshift_crd_test.go

```diff
@@ -704,10 +705,11 @@ func TestOshiftHTTPRuleReencryptWithDestinationCA(t *testing.T) {
 	}, 60*time.Second).Should(gomega.Equal(defaultHostname))
 	VerifySniNode(g, sniVS)
 	g.Eventually(func() bool {
-		return sniVS.PoolRefs[0].SniEnabled
+		return sniVS.PoolRefs[0].SniEnabled != nil &&
```

Comment:
@swathinsankaran  Isn't the **SniEnabled** field supposed to be a boolean?

Reply:
You're right. I have reverted this change. Thanks